fe/fe-core/src/main/java/org/apache/doris/datasource/property/constants/S3Properties.java

```diff
@@ -334,6 +335,18 @@ public static Cloud.ObjectStoreInfoPB.Builder getObjStoreInfoPB(Map<String, Stri
             // S3 Provider properties should be case insensitive.
             builder.setProvider(Provider.valueOf(properties.get(S3Properties.PROVIDER).toUpperCase()));
         }
+
+        if (properties.containsKey(PropertyConverter.USE_PATH_STYLE)) {
+            String value = properties.get(PropertyConverter.USE_PATH_STYLE);
+            Preconditions.checkArgument(!Strings.isNullOrEmpty(value), "use_path_style cannot be empty");
+            Preconditions.checkArgument(value.equalsIgnoreCase("true")
+                    || value.equalsIgnoreCase("false"), "Invalid use_path_style value: %s", value);
+            if (properties.get(PropertyConverter.USE_PATH_STYLE).equalsIgnoreCase("true")) {
+                builder.setUsePathStyle(true);
+            } else {
+                builder.setUsePathStyle(false);
+            }
```

Comment:
```suggestion
            builder.setUsePathStyle(value.equalsIgnoreCase("true"));
```

Reply:
maybe alter set `use_path_style` from true to false